Rubrik/Public/Move-RubrikMountVMDK.ps1

```diff
@@ -133,11 +145,12 @@ function Move-RubrikMountVMDK
         {
           # We just want the very last part of the link, which contains the ID value
           $MountID = $link.href.Split('/')[-1]
+          Write-Verbose -Message "Found Live Mount ID $MountID"
         }
       }
 
-      Write-Verbose -Message 'Gathering details on the Live Mount'
-      $MountVM = Get-RubrikVM -id (Get-RubrikMount -id $MountID).mountedVmId -PrimaryClusterID local
```

Comment:
Passing -PrimaryClusterID along with -id throws an error. It works if -PrimaryClusterID is used along with -Name, but as it was it would always fail.